browser-test/src/support/index.ts

```diff
@@ -12,10 +12,16 @@ export const startSession = async () => {
   return { browser, page }
 }
 
-export const logout = async (page: Page) => page.goto('/logout')
+export const endSession = async (browser: Browser) => {
+  browser.close()
+}
+
+export const logout = async (page: Page) => {
+  // TODO: add logout button to applicant page and use that
+  page.goto('/logout')
+}
 
 export const loginAsAdmin = async (page: Page) => {
-  await page.goto(BASE_URL + '/loginForm')

```

Comment:
question: what will happen if we try  to login from a different page?

Reply:
If we are on a different page, this doesn't work. But we are trying to move away from using URLs to navigate in our browser tests and purely use the UI. As long as we properly log out (which is a prerequisite to log in as admin), we will be redirect to login page so I think removing this should be fine.